### PR TITLE
Redux Redux Patch

### PR DESCRIPTION
**Pull request by mhluongo**
Attempt to fix a couple issues in https://github.com/tahowallet/extension/pull/3757

Latest build: [extension-builds-3758](https://github.com/tahowallet/extension/suites/32399595008/artifacts/2358101407) (as of Tue, 24 Dec 2024 03:37:53 GMT).

### CODE REVIEW

File: background/features.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export function isEnabled(
   }
 
   if (checkBrowserStorage) {
-    const state = "" as string // localStorage.getItem(flagName)
+    const state = localStorage.getItem(flagName)
```
Comment (Shadowfiend) [inline background/features.ts:76]: `localStorage` and `window` aren't available in service workers, that's how we got ourselves here unfortunately :grimacing:

File: background/services/abilities/index.ts [hunk 2]
```diff
@@ -196,7 +196,7 @@ export default class AbilitiesService extends BaseService<Events> {
   }
 
   async fetchAbilities(): Promise<void> {
-    // localStorage.setItem(this.ABILITY_TIME_KEY, Date.now().toString())
+    localStorage.setItem(this.ABILITY_TIME_KEY, Date.now().toString())
```
Comment (Shadowfiend) [inline background/services/abilities/index.ts:199]: Oh also we're not using this whole service at the moment. See line 155, which is where the magic would usually happen <_<

File: background/features.ts (hunk 1 above)
Comment (mhluongo) [reply background/features.ts:76]: I see :upside_down_face:

**Comment by mhluongo** [comment]
sadpanda.gif

**Pull request closed without merging**